### ISSUE TITLE
Run tests on net462

### ISSUE THREAD

**Issue by deleteLater** [featbit/featbit-dotnet-sdk#16]
We should run our tests on both `net462` and `net6.0`, currently we only run these tests on `net6.0`.

https://github.com/featbit/featbit-dotnet-sdk/blob/016f87b8ec0dae3ee34725d5534e12137c73aa69/tests/FeatBit.ServerSdk.Tests/FeatBit.ServerSdk.Tests.csproj#L4

See #15

**Comment by krokofant** [comment]
I imagine the job needs to target the `windows-latest` runner and use the [vstest-action](https://github.com/microsoft/vstest-action). Since the test target is already net6.0 I understand that there might be compatibility issues if the target is changed as-is since unsupported APIs might be used in the test setup.

**Comment by deleteLater** [comment]
Yeah, to run the tests on `net462`, we do need to change some of the test code
